pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -224,6 +224,73 @@ def _set_dimensional_parameters(self):
             "Initial concentration in electrolyte [mol.m-3]"
         )
 
+        # mechanical prapmeters
+        if isinstance(self, pybamm.particle_cracking.BaseCracking):
```

Comment:
replace this with something like `if self.options["particle cracking"] :` and update the options setter at the end of the file

Reply:
Done